packages/protocol/contracts/governance/ReleaseGold.sol

```diff
@@ -477,6 +477,18 @@ contract ReleaseGold is UsingRegistry, ReentrancyGuard, IReleaseGold, Initializa
     getLockedGold().withdraw(index);
   }
 
+  /**
+   * @notice Funds a signer address so that transaction fees can be paid.
+   * @param signer The signer address to fund.
+   */
+  function fundSigner(address payable signer) private {
+    // Fund signer account with 1 cGLD.
+    uint256 value = 1 ether;
+    totalWithdrawn = totalWithdrawn.add(value);
```

Comment:
Perhaps unintuitively, I don't think we want to update `totalWithdrawn` here. Making it 1 while `getCurrentReleasedTotalAmount` is still 0 will cause reverts whenenever we try `releasedAmount.sub(totalWithdrawn)` in `withdraw` and `refundAndFinalize`. 

I believe this will break the revoking flow - if `totalWithdrawn` is set to 1, but this contract gets revoked, our `releasedBalanceAtRevoke` will be 0 since the ratio `periodsSinceStart/numReleasePeriods` will be 0. Then in `refundAndFinalize` this line:
`uint256 beneficiaryAmount = revocationInfo.releasedBalanceAtRevoke.sub(totalWithdrawn);` will always revert, locking the funds. 

I'd suggest not updating `totalWithdrawn` and just treating this like it reduces the grant by 1CG, which it effectively does.

Let me know if that makes sense to you - still thinking out the proposed solution to make sure it doesn't introduce any other issues

Reply:
Makes total sense, nice catch.